### PR TITLE
fix(location): fix location issue.

### PR DESCRIPTION
**Pull request by Enuviel**

### CODE REVIEW

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

File: AndroidSDKCore/src/main/java/com/leanplum/internal/ActionManager.java [hunk 1]
```diff
@@ -68,29 +69,21 @@ public static synchronized ActionManager getInstance() {
...
-            googleApiClientClass.getMethod("isConnected").getModifiers())
-            && Class.forName("com.google.android.gms.location.LocationServices") != null) {
-          // Reflection here prevents linker errors
-          // in Google Play Services is not used in the client app.
-          return (LocationManager) Class
-              .forName("com.leanplum.LocationManagerImplementation")
-              .getMethod("instance").invoke(null);
-        }
+        // Reflection here prevents linker errors
+        // in Google Play Services is not used in the client app.
```
Comment (benmarten) [inline AndroidSDKCore/src/main/java/com/leanplum/internal/ActionManager.java]: if ...

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Pull request merged**